corehq/apps/app_manager/models.py

```diff
@@ -1454,7 +1454,7 @@ def wrap(cls, data):
                     data[media_attr] = new_media
                 elif isinstance(old_media, dict):
                     # Once the media has localized data, discard the dummy key
-                    if 'default' in old_media and len(old_media.keys()) > 1:
+                    if 'default' in old_media and len(list(old_media.keys())) > 1:
```

Comment:
should use `len(old_media)`, which is more efficient because it doesn't need to construct a list just to get the length